com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/CopybookServiceImpl.java

```diff
@@ -52,6 +46,8 @@
   private static final String COPYBOOK_FOLDER_NAME = ".copybooks";
   private final DataBusBroker dataBus;
   private List<WorkspaceFolder> workspaceFolders;
```

Comment:
Seems like you don't need to store the list of `WorkspaceFolder` at all - you convert them on the initialization and then don't use anymore

Reply:
Correct, I will simplify the chain to create the list of paths